sched/definitions.go

```diff
@@ -17,3 +17,33 @@ type Task struct {
 	Command    []string // Argv to run
 	SnapshotId string
 }
+
+type JobState int
+
+const (
+	// An unambiguous 0-value.
+	UNKNOWN JobState = iota
```

Comment:
I don't think the way the system is designed we can tell between UNKNOWN and PENDING.  Sagas are only created and queryable once running